### PR TITLE
Fixes and tweaks

### PR DESCRIPTION
**Pull request by mattleff**
Fixes https://trello.com/c/iIEHxOs4/156-mobile-menu-issues, https://trello.com/c/pLECrv2Z/164-fcbh-disclaimer, https://trello.com/c/IkxP9L4P/138-newsletter-signup-name-field, https://trello.com/c/QOnFZZbf/132-real-teaser-screenshots, https://trello.com/c/OQKCAzTd/141-cross-fade-features-slider-slides.

### CODE REVIEW

File: src/containers/home.tsx [hunk 1]
```diff
@@ -11,147 +11,36 @@ import CardBibleChapter from '@components/molecules/card/bibleChapter';
...
-			cta: (
-				<FormattedMessage
-					id="homePage__feature5Cta"
-					defaultMessage="Find Content to Download"
-				/>
-			),
-			url: makeSermonListRoute(languageRoute),
-		},
-	];
+	const features = getAppFeatures(languageRoute);
```
Comment (narthur) [inline src/containers/home.tsx:43]: Nice!

File: src/lib/getAppFeatures.tsx [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+					id="app__features5Cta"
+					defaultMessage="Find Content to Download"
+				/>
+			),
+			url: makeSermonListRoute(languageRoute),
+			backgroundColor: '#CDD9E3',
+		},
+	].map((item, index) => ({
+		...item,
+		image: `/img/features/${index + 1}.png`,
```
Comment (narthur) [inline src/lib/getAppFeatures.tsx:134]: Clever. :D If you reversed lines 133 and 134, then later on we could override the image on a per-entry basis.

**Review by narthur — APPROVED** [review]
Great work!

File: src/lib/getAppFeatures.tsx (hunk 2 above)
Comment (mattleff) [reply src/lib/getAppFeatures.tsx:133]: c3002bf

File: src/components/organisms/slider.module.scss [hunk 3]
```diff
@@ -76,6 +79,23 @@
...
+			position: absolute;
+			top: 0;
+			left: 0;
+			right: 0;
+			bottom: 0;
+			transition: opacity $transition-medium;
+			opacity: 0;
+		}
+	}
+	$slideNums: 0, 1, 2, 3, 4;
```
[the remark this answers is not in the document]
Comment (mattleff) [reply src/components/organisms/slider.module.scss]: c3002bf

**Pull request merged**